homeassistant/components/media_player/kodi.py

```diff
@@ -155,7 +150,17 @@ def media_duration(self):
     def media_image_url(self):
         """ Image url of current playing media. """
         if self._item is not None:
-            return _get_image_url(self._item['thumbnail'])
+            return self._get_image_url()
+
+    def _get_image_url(self):
+        """ Helper function that parses the thumbnail URLs used by Kodi. """
+        url_components = urllib.parse.urlparse(self._item['thumbnail'])
+        
+        if url_components.scheme == 'image':
+            return\
```

Comment:
Sure. I thought you din't like string format. Don't know where I got that from ☺

I'll do the host URL stuff in a different PR.